src/adagrams.js

```diff
@@ -1,15 +1,133 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const numTiles = 10;
+  const lettersInHand = [];
+  const currentLetterPool = [...listOfLetters]
+  for (let i = 0; i < numTiles; i++) {
+    // 98 letters, i want index from 0 to 97 inclusive
+    let randIndex = Math.floor(Math.random() * currentLetterPool.length);
+    let randLetter = currentLetterPool[randIndex];
```

Comment:
Since neither of the variables get re-assigned in this for loop after lines 7-8, we should use `const` instead. Prefer to always use `const` where possible.

```suggestion
    const randIndex = Math.floor(Math.random() * currentLetterPool.length);
    const randLetter = currentLetterPool[randIndex];
```